### DISCUSSION TITLE
"synchronization file cannot be opened"

### DISCUSSION THREAD

**Discussion by tisquinn** [sumatrapdfreader/sumatrapdf#2741] [Q&A] [answered]
Hi! I just moved to a new laptop and am having some trouble with Sumatra. When I compile a pdf using latex using Sublime Text, sumatra is set to open the new pdf with synchronization. When it opens automatically (i.e., when prompted by sublime text), I get an error saying "synchronization file cannot be opened." If I close the file within Sumatra and then directly open the pdf with sumatra, the synchronization works great. Any idea how I can get it to work when first opened? Thanks!

Quinn

**Comment by GitHubRulesOK** [comment, marked as answer]
see https://github.com/sumatrapdfreader/sumatrapdf/issues/2642
The issue is still open for recent 64bit installs, 3.4.6 should work or try latest 3.5.xxx pre-release however it seems to be variable after 3.3.3 
 if using installed but possibly not when using portable ?
Check your version and try not to use any -inverse-search in the call but each configuration may be different.

**Comment by tisquinn** [comment]
Got it! Sorry—I didn't realize it was the same issue. I misread the opening post about it being a zipped file.... Anyways, my illiteracy aside, switching to the portable version fixed the problem as you suggested. Thanks so so much!!!!

**Comment by ExpPhysU** [comment]
Just an observation. I had a working setup with Winedt 10.3 and SumatraPDF 3.3.3. I tried to install version 3.4.6. with the installer version.
- First issue: It did not honor existing paths and installed to a new location, breaking the link to WinEdt
- Second issue: Reinstallation caused problems with the lib....pdf dll (I do not recall the exact name)
- Third issue: When I had a successful install of SumatraPDF, SumatraPDF complained that there was no information on the source (remember, I did not change WinEDT, the files worked with the previous version.

I then got fed up, uninstalled (or better deleted because the uninstaller did not work properly) version 3.4.6. and reinstalled 3.3.3.: Everything worked as before.

Finally I decided to download the portable version of 3.4.6. I unzipped it into another private folder and set the PDF-viewer in WinEdt to the new location. Everything worked. 8Short comment: In WinEdt you can define up to 4 viewers and switch them in the execution modes panel with a click)

My recommendation: Avoid the installer and use the portable version!

**Comment by note286** [comment]
I have done many tests, and many situations that would have reported `synchronization file cannot be opened` errors before, can be avoided by using `-synctex=-1` not `-synctex=1`.

**Reply by GitHubRulesOK** [reply]
several variations on synctex flag will work as not name.synctex.gz (gz compressed) it seems to be only installed (not portable) and gz compressed (not other synctex files such as uncompressed) that affect some (if not most) of those users.

**Reply by linbdnil** [reply]
I try it, and it works, thanks!

**Reply by 6NiceRice9** [reply]
-synctex=-1 works for me aswell, thank you!
